pkg/util/pravegacluster.go

```diff
@@ -81,8 +82,32 @@ func ConfigMapNameForSegmentstore(clusterName string) string {
 	return fmt.Sprintf("%s-pravega-segmentstore", clusterName)
 }
 
-func StatefulSetNameForSegmentstore(clusterName string) string {
-	return fmt.Sprintf("%s-pravega-segmentstore", clusterName)
+//function to check if the version is below 0.7 or not
+func IsVersionBelow07(ver string) bool {
+	first3 := strings.Trim(ver, "\t \n")[0:3]
+	if first3 == "0.6" || first3 == "0.5" || first3 == "0.4" || first3 == "0.3" || first3 == "0.2" || first3 == "0.1" {
+		return true
+	}
+	return false
+}
+
+func IsClusterUpgradingTo07(p *api.PravegaCluster) bool {
+	/*this check is to avoid creation of a new segmentstore when the CurrentVersionis 06 and target version is 07
```

Comment:
Please provide appropriate comments. "This check is to determine if we are upgrading from a version below 0.7 to version 0.7 or later." Also comments should be provided above the function definition and not inside. Lets use standard coding conventions please.

Reply:
done